client/client.go

```diff
@@ -963,3 +984,93 @@ func (r *NotaryRepository) DeleteTrustData() error {
 	}
 	return nil
 }
+
+// ListRootCerts returns current trusted root certificates.
+// (In particular, not old certificates which we have already
+// rotated from but are still using for signing to allow rollover,
+// even if they are still trusted locally.)
+func (r *NotaryRepository) ListRootCerts() ([]*x509.Certificate, error) {
+	_, err := r.Update(false)
+	if err != nil {
+		return nil, err
+	}
+
+	keyIDs := r.tufRepo.Root.Signed.Roles[data.CanonicalRootRole].KeyIDs
```

Comment:
Yes.  Also, hm... actually looking at this piece, it looks like below where it checks the cert store may not work?  We can't count on the cert store to be updated/cleaned up after an `Update`, right?  Since that only happens currently on bootstrap, and won't happen if we download another root after bootstrap.